### ISSUE TITLE
This problem after input of the password

### ISSUE THREAD

**Issue by rubenSaro** [NrgXnat/oasis-scripts#10]
HTTP/1.1 200 OK
Set-Cookie: SESSION_EXPIRATION_TIME="1596086146219,900000"; Version=1; Path=/
Date: Thu, 30 Jul 2020 05:15:46 GMT
Accept-Ranges: bytes
Server: Noelios-Restlet-Engine/1.1.10
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
X-Frame-Options: SAMEORIGIN
Content-Security-Policy: frame-ancestors 'self'
Content-Length: 0

**Comment by sarahkeefe** [comment]
Hi, typically when you see that output it means that there was a successful request made to the server, so things should be working. But there's a possibility something changed on the server and I need to update the authentication part of the scripts. Which script are you using? Are the files downloading for you at all, and are you getting any other output? Can you copy and paste the full command you are using?

**Comment by solisoares** [comment]
Hi, I am facing the same issue while trying to run any of the `download_oasis_freesurfer_*.sh` files.

I created a XNAT account, but I think is has to do with the fact that I currently do not have access granted by the form provided in the OASIS website.

I am waiting to get my access granted and I'll let you know any updates

**Comment by solisoares** [comment]
Sorry for the late response, but, even with access granted I was having the issue.  
At my organization we decided to try another dataset for our purpose, but if we would keep yours we'd have to solve this after all.

Thank you for the project, it would be way easier to download the images.

**Comment by sarahkeefe** [comment]
Hi, can you send some screenshots of what input you are trying and what the error message looks like? 

You might also try the XNAT "cache refresh" feature. Click on your username at the top of the screen:
![image](https://user-images.githubusercontent.com/264991/161564943-823a6579-ab9c-43fc-a16c-67b7c9321163.png)

Then go to "Manage Cached Resources" on that page:
![image](https://user-images.githubusercontent.com/264991/161565213-49f30f1f-18dd-4b12-8ffa-3aa5a00f94a4.png)

Then click on the button to "Reset Access Cache":
![image](https://user-images.githubusercontent.com/264991/161565334-0e450a30-f4df-47c7-a785-7418359f3f03.png)

**Comment by solisoares** [comment]
Hi sarah.
Actually, after your message today I decided to try it again, and apparently it worked.

With `./download_oasis_freesurfer.sh <input_file.csv> <directory_name> <xnat_central_username>` I used the csv provided as an example. 
![image](https://user-images.githubusercontent.com/77312190/161615862-56981c0e-1a40-409d-a83c-5ef957d673d3.png)

And i got all the respective folders according to this csv.

For the folder `OAS30001_MR_d0129/` I got:
![image](https://user-images.githubusercontent.com/77312190/161617053-69b1ff65-b898-4d66-9623-90f148a8db5f.png)

I think this is it, right? I inspected the contents inside these folders and it seemed ok to me.

When I got access I tried to do this in the same day. And was only downloading an **empty zip file**.
Now, there is no empty file and I can in fact see the downloading happening in my terminal window.

So, maybe when i first tried it was too early for the system allow me and I should have waited a little bit more.

**Comment by sarahkeefe** [comment]
Yes, that looks like it worked, that's great. Yes, you may have tried too early before and the cache hadn't updated yet.
